lib/msf/ui/console/command_dispatcher/db.rb

```diff
@@ -47,13 +47,9 @@ def commands
                 "db_export"     => "Export a file containing the contents of the database",
                 "db_nmap"       => "Executes nmap and records the output automatically",
                 "db_rebuild_cache" => "Rebuilds the database-stored module cache",
-                "test_data_service_host" => "Blah",
-                "test_data_service_loot" => "Blah",
-                "perf_test_data_service_loot" => "REMOVEME - Run a performance test against the loot data service",
-                "add_data_service" => "Blah",
-                "list_data_services" => "Blah",
-                "set_data_service" => "Blah",
-                "nl_search" => "Blah"
+                "add_data_service" => "Adds a data service to metasploit",
+                "list_data_services" => "List data services added to metasploit",
```

Comment:
We should probably change this to just a `data_services` command and add flags for adding, listing, setting, deleting, etc. This will keep it more in line with other commands and not clog up the basic command listings with each action.

If we don't do it in this particular PR we should at least create a backlog ticket for it and do it before merging to master just so people don't get used to a command that is eventually going to change.

Reply:
that's easy I'll do it here